modules/server_new/src/main/scala/observe/server/odb/OdbProxy.scala

```diff
@@ -140,8 +143,15 @@ object OdbProxy {
     override def stepStartObserve(obsId: Observation.Id): F[Boolean] =
       false.pure[F]
 
-    override def datasetStartExposure(obsId: Observation.Id, fileId: ImageFileId): F[Boolean] =
-      true.pure[F]
+    override def datasetStartExposure(
+      obsId:  Observation.Id,
+      fileId: ImageFileId
+    ): F[RecordDatasetMutation.Data.RecordDataset.Dataset] =
+      Sync[F]
+        .delay(scala.util.Random.between(1L, Long.MaxValue))
```

Comment:
You can use
https://typelevel.org/cats-effect/docs/std/random

Reply:
I tried after reading this comment, but for this particular use the current solution is simpler.